contracts/colony/Colony.sol

```diff
@@ -196,6 +196,13 @@ contract Colony is ColonyStorage, PatriciaTreeProofs {
     ERC20Extended(token).mint(address(this), _wad); // ignore-swc-107
   }
 
+  function mintTokensFor(address _guy, uint _wad) public
```

Comment:
Is there not a worry this could be used to pay out tokens without paying the network fee?

Reply:
Not particularly -- when creating a colony normally, the founder is the token owner until they manually transfer it to the colony. So if the founder can mint tokens for free, it's not clear why they would give the token ownership to the colony if that ability would be permanently lost.